lib/cartopy/geodesic/__init__.py

```diff
@@ -0,0 +1,18 @@
+# (C) British Crown Copyright 2011 - 2014, Met Office
```

Comment:
I think this is what @QuLogic meant anyway, but just to be on the safe side... if this is a new file then it should just be `2015` and not a range, e.g. `2011 - 2015`.